dbld/images/helpers/dependencies.sh

```diff
@@ -0,0 +1,145 @@
+}
+
+function install_mongoc() {
+    MONGOC_VERSION=1.13.0
+    download_target "https://github.com/mongodb/mongo-c-driver/releases/download/${MONGOC_VERSION}/mongo-c-driver-${MONGOC_VERSION}.tar.gz" /tmp/mongo-c-driver.tar.gz
+    cd /tmp/
+    tar xvf /tmp/mongo-c-driver.tar.gz
+    cd /tmp/mongo-c-driver-${MONGOC_VERSION}
+    mkdir cmake-build/
+    cd cmake-build/
+    cmake -DENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF ..
+    make
+    make install
+    rm -rf /tmp/mongo-c-driver.tar.gz /tmp/mongo-c-driver-${MONGOC_VERSION}
+}
+
+function install_rabbitmqc() {
+    RABBITMQC_VERSION=0.9.0
+    download_target "https://github.com/alanxz/rabbitmq-c/archive/v${RABBITMQC_VERSION}.tar.gz" /tmp/rabbitmq-c.tar.gz
+    cd /tmp/
+    tar xfz /tmp/rabbitmq-c.tar.gz
+    cd /tmp/rabbitmq-c-${RABBITMQC_VERSION}/
+    mkdir cmake-build/
+    cd cmake-build/
+    cmake ..
+    make
+    make install
+    rm -rf /tmp/rabbitmq-c.tar.gz /tmp/rabbitmq-c-${RABBITMQC_VERSION}/
+}
+
+function install_riemann() {
+    RIEMANN_VERSION=1.10.3
+    download_target "https://github.com/algernon/riemann-c-client/archive/riemann-c-client-${RIEMANN_VERSION}.tar.gz" /tmp/riemann.tar.gz
+    cd /tmp/
+    tar xfz /tmp/riemann.tar.gz
+    cd /tmp/riemann-c-client-riemann-c-client-${RIEMANN_VERSION}
+    autoreconf -i
+    ./configure
+    make
+    make install
+    rm -rf /tmp/riemann.tar.gz /tmp/riemann-c-client-riemann-c-client-${RIEMANN_VERSION}
+}
+
+function download_target() {
+    target=$1
+    output=$2
+    wget --no-check-certificate $target --output-document=$output
+}
+
+function filter_packages_by_platform {
+    FILENAME=$1
+    OS_GROUP=$(echo ${OS_PLATFORM} | cut -d"-" -f1)
+    grep -v "#" ${FILENAME} | grep -e "${OS_PLATFORM}" -e "${OS_GROUP}[^-]" | cut -d"[" -f1
+}
+
+function install_apt_packages {
+    apt-get update -qq -o Acquire::CompressionTypes::Order::=gz
+    filter_packages_by_platform /helpers/packages.manifest | xargs apt-get install --no-install-recommends --yes
+}
+
+function install_yum_packages {
+    filter_packages_by_platform /helpers/packages.manifest | xargs yum install -y
+}
+
+function install_pip_packages {
+    if [ "${OS_PLATFORM}" == "centos-6" ]; then
+        filter_packages_by_platform /helpers/pip_packages.manifest | xargs pip install -U
+    else
+        python -m pip install --upgrade pip
+        filter_packages_by_platform /helpers/pip_packages.manifest | xargs python -m pip install -U
+    fi
+}
+
+function enable_dbgsyms {
+    echo "deb http://ddebs.ubuntu.com $(lsb_release -cs) main restricted universe multiverse
+deb http://ddebs.ubuntu.com $(lsb_release -cs)-updates main restricted universe multiverse
+deb http://ddebs.ubuntu.com $(lsb_release -cs)-proposed main restricted universe multiverse" | \
+    tee -a /etc/apt/sources.list.d/ddebs.list
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 428D7C01 C8CAB6595FDFF622
+    apt-get update
```

Comment:
syncronise it with `/helpers/dependencies.sh install_apt_packages` (apt-get update)